### PR TITLE
feat: Add examples for updating metadata based features on workbench instances.

### PR DESCRIPTION
**Pull request by bcreddy-gcp**
## Description

Fixes #<ISSUE-NUMBER>

Note: If you are not associated with Google, open an issue for discussion before submitting a pull request.

## Checklist

**Readiness**

- [*] Yes, **merge** this PR after it is approved
- [ ] No, don't **merge** this PR after it is approved

**Style**

- [*] My sample follows the rules described for Terraform in the [Effective Samples style guide](https://googlecloudplatform.github.io/samples-style-guide/)
- [*] My sample follows the other requirements and best practices in the [Contributing
guide](https://github.com/terraform-google-modules/terraform-docs-samples/blob/main/CONTRIBUTING.md#other-requirements-and-best-practices)

**Testing**

- [*] I have performed tests described in the [Contributing guide](https://github.com/terraform-google-modules/terraform-docs-samples/blob/main/CONTRIBUTING.md):

   - [*] **[Tests](https://github.com/terraform-google-modules/terraform-docs-samples/blob/main/CONTRIBUTING.md#set-up-the-test-environment)** pass: `terraform apply`
   - [*] **[Lint](https://github.com/terraform-google-modules/terraform-docs-samples/blob/main/CONTRIBUTING.md#linting-and-formatting)** pass: `terraform fmt` check

**Intended location**

- [*] Yes, this sample will be (or already is) included on cloud.google.com
      Location(s):

- [ ] No, this sample won't be included on cloud.google.com
      Reason:

**API enablement**

- [*] If the sample needs an API enabled to pass testing, I have added the service to the [Test setup file](https://github.com/terraform-google-modules/terraform-docs-samples/blob/main/test/setup/main.tf)

**Review**

- [*] If this sample adds a new directory, I have added codeowners to the [CODEOWNERS file](https://github.com/terraform-google-modules/terraform-docs-samples/blob/main/.github/CODEOWNERS)

### CODE REVIEW

**Comment by bcreddy-gcp** [comment]
snippet-bot:force-run

File: vertex_ai/workbench_create_metadata/main.tf [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+# [START aiplatform_workbench_create_metadata]
+resource "google_workbench_instance" "default" {
+  name     = "workbench-instance-example"
+  location = "us-central1-a"
```
Comment (glasnt) [inline vertex_ai/workbench_create_metadata/main.tf:21]: These samples look good, just a question I had here: is "location" in this case the zone, as opposed to (on other resources I've seen) the region? The tf resource docs aren't clear on this separation. Will this be documented in the docs this sample will pair with?

Comment (bcreddy-gcp) [reply vertex_ai/workbench_create_metadata/main.tf:21]: Resources are usually regional/zonal based on their end points. I don't think I've seen documentation mentioning this specifically anywhere. Usually this info is inferred from the values.

**Comment by glasnt** [comment]
requires review from @terraform-google-modules/dee-data-ai

**Review by gericdong — APPROVED** [review]

**Pull request merged**